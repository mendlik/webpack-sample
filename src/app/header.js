import style from './header.scss';

const header = () => {
  const element = document.createElement('header');
  const text = document.createTextNode('Header');
  element.appendChild(text);
  element.classList.add(style.content);
  return element;
};

export default header;
