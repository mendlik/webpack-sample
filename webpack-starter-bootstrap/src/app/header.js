import html from './header.html';
import { parseHtmlNode } from './util';

const header = () =>
  parseHtmlNode(html);

export default header;
