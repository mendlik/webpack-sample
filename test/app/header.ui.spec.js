import header from '../../src/app/header';

describe('Header', () => {
  it('should create header element', () => {
    const element = header();
    expect(element).toMatchSnapshot();
  });
});
