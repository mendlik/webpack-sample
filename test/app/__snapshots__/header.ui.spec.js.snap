// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should create header element 1`] = `
<header
  class="undefined"
>
  Header
</header>
`;
